bin/lookup_google_apps_email.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ -e /opt/virtualenvs/google_legacy_oauth ]; then
+    . /opt/virtualenvs/google_legacy_oauth/.venv/bin/activate
```

Comment:
Minor point but you could save some unnecessary work done by `activate` by using the python executable directly, e.g.
```bash

PYTHON=$(which python)
if [ -e /opt/virtualenvs/google_legacy_oauth ]; then
  PYTHON="/opt/virtualenvs/google_legacy_oauth/.venv/bin/python"
fi

$PYTHON /data/mysociety/bin/lookup_google_apps_email.py $1
```

(In this case it removes the need for the `env` change to the shebang in the Python script, but having it there is good practice anyway.)